.github/workflows/release.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           submodules: recursive
       - name: Install cargo-dist
         run: |
-          cargo install --locked \
+          cargo install \
             --git "https://github.com/webern/cargo-dist" \
```

Comment:
Locking dependencies in these cases seems like a good idea to me. Should we really be removing this?